HW11/helicopter.py

```diff
@@ -0,0 +1,43 @@
+from HW11.vehicle import Vehicle
+
+
+# Define the Helicopter class, which inherits from the Vehicle class
+# Inheritance
+
+class Helicopter(Vehicle):
+
+    def recharge(self):
+        pass
```

Comment:
what is it for?

Reply:
ops, I wanted to use abstract method for this, but forgot realize that moment